app/src/main/java/org/oppia/android/app/databinding/StateAssemblerPaddingBindingAdapters.java

```diff
@@ -105,6 +111,11 @@ private static void setPaddingIfApplicable(
       float paddingEnd,
       float paddingBottom
   ) {
+    if (isRtlLayout(view)) {
```

Comment:
Ditto here (but for paddingLeft/paddingRight).

Reply:
Done,